torch/csrc/jit/passes/create_functional_graphs.cpp

```diff
@@ -294,6 +296,38 @@ struct MutationRemover {
         mutated_value->node(), mutating_op);
   }
 
+  bool tryMakeUnaliasedIfOutputAndMutationAtomic(
+      Value* mutated_value,
+      Node* mutating_op) {
+    // if cond:
```

Comment:
as discussed offline, can we change this comment to make it more clear that `x` is a single symbol?